### ISSUE TITLE
Update docs for officially supported helm charts 

### ISSUE THREAD

**Issue by Hafflgav** [camunda-cloud/camunda-cloud-documentation#609]
Since @Zelldon is currently working on officially supporting the helm charts for camunda cloud self-managed the documentation needs to be updated as well. 
The goal is to update https://docs.camunda.io/docs/self-managed/zeebe-deployment/kubernetes/index/

Related epic: 
https://github.com/camunda-community-hub/camunda-cloud-helm/issues/123

**Comment by Hafflgav** [comment]
What do you think about changing the structure of the content @Zelldon? 
I am thinking about combining Kubernetes Overview & Helm overview so we would end up with one hierarchy less. 

![image](https://user-images.githubusercontent.com/30292227/154100427-e876ae67-9d20-454d-af22-684bed2fd3a0.png)

**Comment by Zelldon** [comment]
Sorry for the late response @Hafflgav 

Yeah I was generally interested in revisting the current structure. For me it would make sense to have instead of 

SelfManaged:
- Zeebe 
  * Docker
  * K8
- Tasklist
- Operate

Something Like:
```
SelfManaged:
* Kubernetes
  * Zeebe
  * Operate
  * Tasklist
* Docker
  * Zeebe
  * Operate
  * Tasklist
```

Would like to hear the opinion of @akeller

**Comment by akeller** [comment]
My understanding of Kubernetes/Helm was we would have a single chart for all the components, so would we need to split out the components under the Kubernetes section individually?

I like your approach here @Zelldon. And I definitely appreciate one less hierarchy as @Hafflgav suggests! Great mindset to bring to these docs!

**Comment by Hafflgav** [comment]
Even though I like the structure proposed by @Zelldon a lot I have problems splitting up the content properly. In my opinion it will be quite an effort to achieve this structure 

Due to this I wanted to propose this structure: 

* Deployment 
   * Kubernetes 
   * Docker 
   * LocalInstallation
* Zeebe  
   * Configuration 
   * Security
   * Operation
* Operate
* Tasklist
* Optimize
* IAM 
* Troubleshooting 

By doing so we will have one chapter for deployment options and the others for configuration options of the different tools.
Any other opinions or remarks @Zelldon, @akeller & @felix-mueller?

**Comment by felix-mueller** [comment]
Like what you are suggesting

**Comment by Hafflgav** [comment]
That is true - But to me it seems that all the other pages do not have a clear separation between the deployment strategy. As far as my (still limited) knowledge goes, we would need to rip them apart and recreate them accordingly so it makes sense. 
Since this seems like a lot of work (and I do have limited capacities) I wanted to find a quicker fix for the first solution. 

But I agree with you -> In the ultimate goal is to have a structure which you have shown above.

**Comment by Zelldon** [comment]
Might be better to do it in iterations then, and open more issues for potential improvements and ideas etc This would also fit better in situations where we have time-constraints and we can might distribute the issues if they are clear enough :) So like first merge the changes you did and then we (me or you or whoever has time and need) can continue to work on them.

**Comment by Hafflgav** [comment]
Agreed! Let's proceed this way. @akeller or @christinaausley will close the PR #614 soon and I will create a new issue for the structure of the self-managed section in our documentation. 
I will ping you once this is done. Somehow splitting up the work seems promising! Happy to help - as always 👍
